src/auth/AuthSessionHandler.cc

```diff
@@ -38,14 +38,3 @@ AuthSessionHandler *get_auth_session_handler(CephContext *cct, int protocol, Cry
   }
   return NULL;
 }
-
-
-void AuthSessionHandler::print_auth_session_handler_stats() {
```

Comment:
> it would remove this purpose too

i did read this comment. and we should remove this comment as well if these member variables are going away.